src/config.coffee

```diff
@@ -413,7 +413,7 @@ class Config
   # * `keyPath` (optional) {String} name of the key to observe. Must be
   #   specified if `scopeDescriptor` is specified.
   # * `optional` (optional) {Object}
```

Comment:
I say go for it.  Let's make the docs for this function 100% correct rather than 75% correct :smile:.